app/helpers/arclight_helper.rb

```diff
@@ -7,8 +7,8 @@ module ArclightHelper
   # @param [SolrDocument]
   def parents_to_links(document)
     safe_join(Arclight::Parents.from_solr_document(document).as_parents.map do |parent|
-      link_to parent.label, solr_document_path(parent.global_id)
-    end, t('arclight.breadcrumb_separator'))
+      link_to  parent.label, solr_document_path(parent.global_id), '</div>'
```

Comment:
It looks like there might be an issue with this syntax.  Should the `</div>` be on the same line as the `link_to`?

Reply:
Hi Jack, it is not popping up a syntax error for me and seems to be working locally but I'll take a look.

Reply:
Arrg, forgot to rebase on master but that's now fixed.

Reply:
Esty helped with the link_to issues. Waiting for travis now.

Reply:
I believe this is resolved.